_conferences/devfest-london-2015.md

```diff
@@ -0,0 +1,8 @@
+---
+name: "GDG DevFest London 2015"
```

Comment:
This should not include the location or the year. Both pieces of information are already included in other information in this file (and thus on the website).